server/apps/scaling-pulsar-smtp/notes.md

```diff
@@ -0,0 +1,5 @@
+* En isolation, supprimer server/data/data-api/src/main/resources et rejouer tous les tests pour voir où ces fichiers manquent
+* Loading
+  * Extraire MailRepositoryFactory et l'implementer pour les différentes implems de MailRepository 
+  * Remplacer l'implem de GuiceMailRepositoryLoader par celle du multibound
+  * Créer les provides into set dans les modules des implem de mail repository (memory et cassandra)
```

Comment:
Remember to remove this when no longer needed (French spotted ;-))

Reply:
fixed up